OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -164,8 +165,11 @@ public void Tick(World world)
 			if (info.ContrailLength > 0)
 				contrail.Update(pos);
 
-			if (ticks++ >= length || (info.Blockable && world.ActorMap
-				.GetUnitsAt(world.Map.CellContaining(pos)).Any(a => a.HasTrait<IBlocksProjectiles>())))
+			if (ticks++ >= length
```

Comment:
it actually shouldn't matter in practice due to order latency, so never mind :)

On 2015-09-05, at 11:30 PM, reaperrr <notifications@github.com> wrote:

> In OpenRA.Mods.Common/Effects/Bullet.cs:
> 
> > @@ -164,8 +165,11 @@ public void Tick(World world)
> >  			if (info.ContrailLength > 0)
> >  				contrail.Update(pos);
> >  
> > -			if (ticks++ >= length || (info.Blockable && world.ActorMap
> > -				.GetUnitsAt(world.Map.CellContaining(pos)).Any(a => a.HasTrait<IBlocksProjectiles>())))
> > +			if (ticks++ >= length
> Why and how?
> 
> ―
> Reply to this email directly or view it on GitHub.
>